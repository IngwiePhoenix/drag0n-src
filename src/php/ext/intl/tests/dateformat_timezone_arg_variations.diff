002+ domingo, 1 de enero de 2012 01:00:00 Hora estándar de Europa central
003+ domingo, 1 de enero de 2012 00:00:00 Hora estándar de Europa occidental
002- domingo%S 1 de enero de 2012 01:00:00 Hora estándar de Europa Central
003- domingo%S 1 de enero de 2012 00:00:00 Hora%S de Europa Occidental
005+ domingo, 1 de enero de 2012 01:00:00 Hora estándar de Europa central
006+ domingo, 1 de enero de 2012 00:00:00 Hora estándar de Europa occidental
007+ domingo, 1 de enero de 2012 01:00:00 Hora estándar de Europa central
005- domingo%S 1 de enero de 2012 01:00:00 Hora estándar de Europa Central
006- domingo%S 1 de enero de 2012 00:00:00 Hora%S de Europa Occidental
007- domingo%S 1 de enero de 2012 01:00:00 Hora estándar de Europa Central