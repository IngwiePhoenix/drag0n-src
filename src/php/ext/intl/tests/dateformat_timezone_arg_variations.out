sábado, 31 de diciembre de 2011 23:00:00 Hora estándar de las Azores
domingo, 1 de enero de 2012 01:00:00 Hora estándar de Europa central
domingo, 1 de enero de 2012 00:00:00 Hora estándar de Europa occidental
sábado, 31 de diciembre de 2011 19:00:00 Hora estándar oriental
domingo, 1 de enero de 2012 01:00:00 Hora estándar de Europa central
domingo, 1 de enero de 2012 00:00:00 Hora estándar de Europa occidental
domingo, 1 de enero de 2012 01:00:00 Hora estándar de Europa central